src/Compilers/CSharp/Test/Syntax/Parsing/InterpolationTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing
+{
+    public class InterpolationTests
+    {
+        [Fact]
+        public void APIBackCompatTest1()
```

Comment:
Thanks for adding those tests. Consider sticking them in one of the other test files (doesn't seem worth adding new files for just this). (same comment for `InterpolatedStringExpressionTests.cs`)

Reply:
so, i would like to keep these files as i think this really is about the InterpolationSyntax api and thus should be in something dedicated to that.  but i don't feel strognly :)

Reply:
That's fine. File-scope the namespaces in those API test files then?

Reply:
sgtm!